tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/service/BrukerServiceImpl.kt

```diff
@@ -132,37 +128,64 @@ class BrukerService(
 		return brukerRepository.upsert(bruker)
 	}
 
-	fun update(bruker: BrukerDbo): Boolean {
-		val hasChanges = fun(bruker: BrukerDbo, newData: BrukerUpsertDbo): Boolean {
-			return bruker.fornavn != newData.fornavn
-				|| bruker.mellomnavn != newData.mellomnavn
-				|| bruker.etternavn != newData.etternavn
-				|| bruker.telefonnummer != newData.telefonnummer
-				|| bruker.epost != newData.epost
-		}
+	fun updateBruker(brukerId: UUID): Boolean {
+		return brukerRepository.get(brukerId)
+			?.let { update(it) }
+			?: throw NoSuchElementException("Bruker med id $brukerId eksisterer ikke")
```

Comment:
Dette var litt forvirrende pga navngivning. Ser ut som man bare oppdaterer en bruker med samme data. Kan kalle `update` for noe som antyder at det er en synkronisering av brukerdata(eller noe sånt)?